### PR TITLE
Irm 10 specific cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Note - SWND 2 sensor on SUMO is at same height as FDCHP for this deployment; added comment in cal csv for LGR038
Sensor heights in LGR cal csvs can be confirmed in attached file.
[METBK_Module_Heights.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/12195471/METBK_Module_Heights.xlsx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Irminger 10 AM Updates](https://trello.com/c/n3g5jRk2/198-irminger-10-am-updates)

File: calibration/METBKA/CGINS-METLGR-00033__20230827.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR033,CC_depth_of_conductivity_and_temperature_measurements_m,1.23,GI01SUMO-00010 METBK1
+LGR033,CC_height_of_air_humidity_measurement_m,5.255,
+LGR033,CC_height_of_air_temperature_measurement_m,5.255,
+LGR033,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,Constant
```
Comment (cooleyky) [inline calibration/METBKA/CGINS-METLGR-00033__20230827.csv:9]: Looks good

File: calibration/METBKA/CGINS-METLGR-00038__20230827.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR038,CC_depth_of_conductivity_and_temperature_measurements_m,1.37,GI01SUMO-00010 METBK2
+LGR038,CC_height_of_air_humidity_measurement_m,5.255,
+LGR038,CC_height_of_air_temperature_measurement_m,5.255,
+LGR038,CC_height_of_windspeed_sensor_above_sealevel_m,5.85,SWND 2 sensor raised to same height as FDCHP
+LGR038,CC_jcool,1,Constant
+LGR038,CC_jwarm,1,Constant
+LGR038,CC_zinvpbl,600,Constant
+LGR038,CC_use_velpt,1,Constant
```
Comment (cooleyky) [inline calibration/METBKA/CGINS-METLGR-00038__20230827.csv:9]: looks good

**Review by cooleyky — APPROVED** [review]
looks good

File: calibration/ADCPSL/CGINS-ADCPSL-21446__20230827.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+21446,CC_scale_factor1,0.45,constant; GI03FLMA-00010
+21446,CC_scale_factor2,0.45,constant
+21446,CC_scale_factor3,0.45,constant
+21446,CC_scale_factor4,0.45,constant
+21446,CC_bin_size,1000,in cm
+21446,CC_dist_first_bin,1647,in cm
+21446,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-21446__20230827.csv:8]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-23381__20230827.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+23381,CC_scale_factor1,0.45,constant; GI03FLMB-00010
+23381,CC_scale_factor2,0.45,constant
+23381,CC_scale_factor3,0.45,constant
+23381,CC_scale_factor4,0.45,constant
+23381,CC_bin_size,1000,in cm
+23381,CC_dist_first_bin,1647,in cm
+23381,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-23381__20230827.csv:8]: looks good

File: calibration/ADCPSN/CGINS-ADCPSN-21654__20230827.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+21654,CC_scale_factor1,0.45,constant; GI01SUMO-00010
+21654,CC_scale_factor2,0.45,constant
+21654,CC_scale_factor3,0.45,constant
+21654,CC_scale_factor4,0.45,constant
+21654,CC_bin_size,1000,in cm
+21654,CC_dist_first_bin,1647,in cm
+21654,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSN/CGINS-ADCPSN-21654__20230827.csv:8]: looks good

File: calibration/METBKA/CGINS-METLGR-00033__20230827.csv (hunk 1 above)
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00033__20230827.csv:9]: looks good

File: calibration/METBKA/CGINS-METLGR-00038__20230827.csv (hunk 2 above)
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00038__20230827.csv:9]: looks good

**Review by cdobs — APPROVED** [review]
looks good to me

**Pull request merged**